### DISCUSSION TITLE
Docs: add some information about how this differs from other benchmarking tools

### DISCUSSION THREAD

**Discussion by jeromatron** [nosqlbench/nosqlbench#2089] [Ideas]
It would be great to have some additional information in the docs on how this differs from other benchmarking tools such as YCSB, cassandra-stress, and tlp-stress.

**Comment by jshook** [comment]
We have a showcase section in the docs that tries to call out the things that are unique to NoSQLBench without a point-by-point style of comparison to other tools.

Much of what nb does happens to be what many other tools don't, but this is more a result of building a tool to fill a need that wasn't being met rather than focusing too much on the state of the other tooling. That said, I do appreciate that users want to know specifically how to compare the tools to make informed decisions. In many ways, comparing nb to other tools can lead to false impressions that they are the same "kind" of tool, but depending on your needs this might not be entirely fair.

For example, you can't do scripting in any of the other tools, which should indicate more that nb is in a separate category of tools that can subsume what the others do. This is in contrast to thinking of scripting as a bullet point feature of all the tools in a category. There are several other facets like this which make authentic comparison more difficult, like real-time controls, discrete metrics collection, dashboard integration, and so on.

Maybe we can add some details in to the show case section of the docs that say something like "This is not possible to do in tools like ..." or "This is not easy to do in .,.." for example. I'm generally much more comfortable talking about the awesome things we can demonstrate than looking closely at other tools for comparisons. After all, NoSQLBench wasn't based on any of them. If the other tools could have done what was needed in terms of accuracy, portability, flexibility, efficiency, and so on, there would be no NoSQLBench.

If someone is interested in doing a point-by-point while keeping the above in mind, we can find a way to put it into the new user view in case it is helpful. I'd generally be more interested in showing examples of awesomeness (that you can't do in any other tool) rather than having a grid-like view of fine points.
